rx/__init__.py

```diff
@@ -3,8 +3,23 @@
 from asyncio.futures import Future as _Future
 from typing import Iterable, Callable, Any, Optional, Union
 
+import os
```

Comment:
Maybe modules from the standard library should be imported first, as suggested by the [python style guide](https://www.python.org/dev/peps/pep-0008/#imports).

Reply:
Alright, I'll fix this.